pkg/pb/orders.proto

```diff
@@ -50,6 +50,36 @@ message OrderLimit {
     node.NodeAddress satellite_address = 11;
 }
 
+// OrderLimitSigning provides OrderLimit signing serialization
+message OrderLimitSigning {
```

Comment:
Could we have a little bit more explanation here also? For example, maybe something like "OrderLimitSigning messages are never directly stored or sent across the network; they are only used when calculating a signature over an OrderLimit. Its purpose is to solidify the format of how we serialize OrderLimits for signing, to handle some backwards compatibility considerations."?